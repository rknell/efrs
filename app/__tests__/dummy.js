/**
 * Seems like a stupid test file but has given me quick validation
 * at times that there is a problem with the test enironment
 */

import React, { Component } from 'react'
import { AppRegistry, Text, View } from 'react-native'
import renderer from 'react-test-renderer'

class myApp extends Component {
  render () {
    return (
      <View><Text>Hello world</Text></View>
    )
  }
}

AppRegistry.registerComponent('myApp', () => myApp)

it('renders at all', () => {
  const tree = renderer.create(<myApp />).toJSON()
  expect(tree).toMatchSnapshot()
})
