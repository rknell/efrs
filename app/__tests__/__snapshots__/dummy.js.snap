// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders at all 1`] = `<myApp />`;
